src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import "./NavBar.css";
+import logo from "../../logo.svg"
+function NavBar() {
+  const [click, setClick] = useState(false);
+
+  const handleClick = () => setClick(!click);
+  return (
+    <>
+      <nav className="navbar">
+        <div className="nav-container">
+          <NavLink exact to="/" className="nav-logo">
+            <img src={logo} alt="MokuMoku" />
+          </NavLink>
+
+          <ul className={click ? "nav-menu active" : "nav-menu"}>
+            <button className="nav-item">
+              <NavLink
+                exact
+                to="/login"
+                activeClassName="active"
+                className="nav-links"
+                onClick={handleClick}
```

Comment:
I think there is no function for this onClick function for NavLink. When we click the NavLink, it will direct to the _/login_